lib/propshaft/assembly.rb

```diff
@@ -18,6 +18,10 @@ def load_path
     @load_path ||= Propshaft::LoadPath.new(config.paths, version: config.version)
   end
 
+  def app_stylesheets_load_path
```

Comment:
Don't think this is the right level to have something like this. Let's keep that in the helper.

Reply:
Right, I've moved it to the helper. Do you think it would be better to memoize it?